### PR TITLE
Add smoke test key to config, but dont use it yet

### PR DESCRIPTION
**Pull request by dr3**
**Overall change:** Add smoke test key to config

**Code changes:**

- Add smoke test key to config

---

- [x] I have assigned myself to this PR and the corresponding issues
- ~[ ] Tests added for new features~
- [ ] Test engineer approval

### CODE REVIEW

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -3,119 +3,188 @@ export default {
...
-        Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
-          ? undefined
-          : '/amharic/articles/c123456abcdo',
-      frontPage: Cypress.env('APP_ENV') === 'live' ? undefined : '/amharic',
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live'
+            ? undefined
+            : '/amharic/articles/c3rykrrvy19o',
+        smoke: true,
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js]:
```suggestion
        smoke: false,
```

File: cypress/support/config/services.js [hunk 2]
```diff
@@ -126,179 +195,281 @@ export default {
...
+        smoke: false,
+      },
+      errorPage404: {
+        path:
+          Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
+            ? undefined
+            : '/naidheachdan/articles/c123456abcdo',
+        smoke: false,
+      },
       frontPage:
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js]: missing smoke here

File: cypress/support/config/services.js [hunk 3]
```diff
@@ -309,341 +480,512 @@ export default {
...
-        Cypress.env('APP_ENV') === 'live' || Cypress.env('APP_ENV') === 'test'
-          ? undefined
-          : '/pashto/articles/c123456abcdo',
-      frontPage: Cypress.env('APP_ENV') === 'live' ? undefined : '/pashto',
+      articles: {
+        path:
+          Cypress.env('APP_ENV') === 'live'
+            ? '/pashto/articles/c70970g2251o'
+            : '/pashto/articles/cng0e8v85eko',
+        smoke: false,
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js]:
```suggestion
        smoke: true,
```

File: cypress/support/config/services.js [hunk 4]
```diff
@@ -309,341 +480,512 @@ export default {
...
     },
   },
   portuguese: {
     font: undefined,
     isWorldService: true,
     pageTypes: {
-      articles: undefined,
-      errorPage404: undefined,
-      frontPage: undefined,
+      articles: { path: undefined, smoke: false },
```
Comment (jamesbhobbs) [inline cypress/support/config/services.js:600]: Why does this not have an article path and a 404 path? It's OOS, but is this is a known issue?

**Review by jamesbhobbs — CHANGES_REQUESTED** [review]

File: cypress/support/config/services.js (hunk 4 above)
Comment (dr3) [reply cypress/support/config/services.js:600]: Ah I might have confused it for a pending one because of the portuguese/brasil logic, ile add it now

Comment (dr3) [reply cypress/support/config/services.js:600]: Added frontpage and article, https://www.test.bbc.com/portuguese/articles/cerj3rk9mmyo is 404'ing for some reason, dispite https://www.test.bbc.com/portuguese/articles/cerj3rk9mmyo.json working through... any ideas @hindsc52?

**Review by jamesbhobbs — APPROVED** [review]

**Pull request merged**